pkg/internal/checkup/executor/executor.go

```diff
@@ -138,13 +134,13 @@ func (e Executor) Execute(ctx context.Context, vmiName, podName, podContainerNam
 		return status.Results{}, err
 	}
 
-	var trafficGeneratorSrcPortStats portStats
+	var trafficGeneratorSrcPortStats PortStats
 	trafficGeneratorSrcPortStats, err = trexClient.GetPortStats(ctx, trafficSourcePort)
 	if err != nil {
 		return status.Results{}, err
 	}
 
-	var trafficGeneratorDstPortStats portStats
+	var trafficGeneratorDstPortStats PortStats
```

Comment:
I think this change belongs to a previous commit.

Reply:
yep. fixed